src/extraction/fathom/ruleset_factory.js

```diff
@@ -247,93 +225,86 @@ export default class RulesetFactory {
    */
   aspectRatio(element) {
     const rect = element.getBoundingClientRect();
+    if (rect.width === 0 || rect.height === 0) {
+      return Infinity;
+    }
     return (rect.width > rect.height) ? (rect.width / rect.height) : (rect.height / rect.width);
   }
 
   /** Give a bonus for elements that have a non-extreme aspect ratio. */
   hasSquareAspectRatio(fnode) {
-    return linearScale(this.aspectRatio(fnode.element), 10, 5) ** this.extremeAspectCoeff;
+    return linearScale(this.aspectRatio(fnode.element), 10, 5);
   }
 
   /**
   * Using coefficients passed into the constructor method, returns a weighted
   * ruleset used to score elements in an HTML document.
+  *
+  * @param {Array[]} An array of [string, number] tuples where the first element
```

Comment:
Is it legal to leave out the param name? In any case, we should either document both params or neither.